src/genie/libs/parser/iosxe/tests/ShowAvcSdServiceInfoSummary/cli/equal/golden_output_2_expected.py

```diff
@@ -0,0 +1,5 @@
+expected_output = {
+    "active_controller": {"status": "not configured"},
+    "device": {"id": "", "ipv4_address": "0.0.0.0", "segment_name": "global"},
+    "sdavc": {"status": "DISCONNECTED"},
```

Comment:
not 100% sure if the `Status` is same with `SD-AVC is disabled`. 
let's take this like below:
```
"status": "DISCONNECTED",
```
and take `SD-VAC is disabled` like below:
```
"sd_vac_status": "disabled",

Reply:
Updated!